src/ldp/representation/RepresentationPreference.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Represents a single preference in a request.
+ */
+export interface RepresentationPreference {
+  /**
+   * The actual preference value.
+   */
+  value: string;
+  /**
+   * How important this preference is in a value going from 0 to 1.
```

Comment:
Can you add an explanation on what it means for weight to be undefined?

Reply:
I'm thinking now to simply make the weight non-optional, forcing parsers to fill in the value (and specifically for us, forcing parsers to enter 1 if the incoming headers have no weight value). No reason to have it optional if it not being present has to be interpreted as a value further down the line anyway.